### PR TITLE
Added Scorm time to lp - Refs #8211

### PR DESCRIPTION
**Pull request by jloguercio**

### CODE REVIEW

File: main/lp/learnpathItem.class.php [hunk 1]
```diff
@@ -3627,8 +3627,18 @@ public function scorm_update_time($total_sec = 0)
             error_log("total_time: $total_time");
         }
 
+        $lp_table = Database::get_course_table(TABLE_LP_MAIN);
+        $lp_id = intval($this->lp_id);
+        $sql = "SELECT * FROM $lp_table
+                    WHERE id = '$lp_id' AND c_id = $course_id";
```
Comment (ywarnier) [inline main/lp/learnpathItem.class.php]: Remove the single quotes around $lp_id -> it is an integer, so don't assign it as if it was a string ;-)

File: app/Migrations/Schema/V111/Version20160705190000.php [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+class Version20160705190000 extends AbstractMigrationChamilo
+{
+    /**
+     * @param Schema $schema
+     * @throws \Doctrine\DBAL\DBALException
+     * @throws \Doctrine\DBAL\Schema\SchemaException
+     */
+    public function up(Schema $schema)
+    {
+        $this->addSql("ALTER TABLE c_lp ADD COLUMN accumulate_scorm_time VARCHAR(5)");
```
Comment (ywarnier) [inline app/Migrations/Schema/V111/Version20160705190000.php]: Defines un varchar(5) aquí pero el entity dice "boolean"...

Comment (jloguercio) [inline app/Migrations/Schema/V111/Version20160705190000.php]: Fixed Here : https://github.com/chamilo/chamilo-lms/pull/1268

File: main/lp/learnpathItem.class.php (hunk 1 above)
Comment (jloguercio) [inline main/lp/learnpathItem.class.php]: Fixed here : https://github.com/chamilo/chamilo-lms/pull/1268

**Pull request merged**